PassiFy/Program.cs

```diff
@@ -16,7 +13,7 @@ static void Main()
         {
             Application.EnableVisualStyles();
             Application.SetCompatibleTextRenderingDefault(false);
-            Application.Run(new Form1());
+            Application.Run(new PasswordForm());

```

Comment:
Give thing meaningful names. Even if the project is simple it is very bad practice to name things TextBox1, Form1, etc.